packages/vue3-vite/src/external/tsconfig-base-aliases.ts

```diff
@@ -0,0 +1,49 @@
+import path from 'path';
+import fs from 'fs';
+
+const offsetToFile = (
+  searchPath: string,
+  configName = 'tsconfig.base.json'
+): string | undefined => {
+  let offset = './';
+  let prevConfigPath = '';
+  // eslint-disable-next-line no-constant-condition
+  while (true) {
+    const configPath = path.join(searchPath, offset, configName);
+    if (fs.existsSync(configPath)) {
+      return offset;
+      // Exit if we've reached the FS root
+    } else if (configPath === prevConfigPath) {
+      return undefined;
+    }
+    offset += '../';
+    prevConfigPath = configPath;
+  }
+};
+
+export const tsconfigBaseAliases = (
```

Comment:
@abemscac I moved this function to the `nx-vue3-vite` package so that we can import it in `vite.config.ts` instead of making a copy for every new app/lib. I had to add the `offsetToFile` function to get the relative path offset to `tsconfig.base.json`, I didn't see a simpler way t do it.

I should probably add some unit tests, but it seems to work for now. I'm going to see if it breaks anything in our project.